app/elements/candidates/invite-candidate-input.html

```diff
@@ -45,8 +50,8 @@
 				<p>{{challenges.0.Name}}</p>
 			</template>
 			<template is="dom-if" if="{{!isSingleChallenge}}">
-				<select id="challenges" on-change="_setChallenge">
-					<option selected disabled>Please select a challenge</option>
+				<select id="challenges" class$={{selectClass}} on-change="_setChallenge">
```

Comment:
https://elements.polymer-project.org/elements/paper-dropdown-menu

if we use this, we shoudn't need any custom validations and css-class hacks